CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/plugins/experimental/TestParametersPlugin.java

```diff
@@ -107,6 +109,7 @@
     @ServiceProvider(service = Plugin.class)
 })
 @Messages("TestParametersPlugin=Test Parameters")
+@PluginInfo(pluginType = PluginType.SELECTION, tags = {"EXPERIMENTAL", "SELECT"})
 public class TestParametersPlugin extends RecordStoreQueryPlugin implements DataAccessPlugin {
```

Comment:
Given it sits in the Developer section of DAV, would it make sense to tag with DEVELOPER? Tagging with SELECT also is a bit confusing to me

Reply:
I'm not sure how or why `SELECT` came into this. I am certain I was meant to put `MODIFY`. I'll have another go and thank you for picking up on this.

Reply:
Fixed